packages/client/components/NewMeetingActions.tsx

```diff
@@ -8,61 +8,76 @@ import StartTeamPromptMutation from '~/mutations/StartTeamPromptMutation'
 import {NewMeetingActions_team} from '~/__generated__/NewMeetingActions_team.graphql'
 import {MeetingTypeEnum} from '~/__generated__/NewMeetingQuery.graphql'
 import useAtmosphere from '../hooks/useAtmosphere'
-import useBreakpoint from '../hooks/useBreakpoint'
 import useMutationProps from '../hooks/useMutationProps'
 import useRouter from '../hooks/useRouter'
 import StartSprintPokerMutation from '../mutations/StartSprintPokerMutation'
 import {Breakpoint} from '../types/constEnums'
-import Icon from './Icon'
+import FlatButton from './FlatButton'
+import FlatPrimaryButton from './FlatPrimaryButton'
 import NewMeetingActionsCurrentMeetings from './NewMeetingActionsCurrentMeetings'
-import PrimaryButton from './PrimaryButton'
 import StyledError from './StyledError'
 
-const newMeetingGridMediaQuery = `@media screen and (min-width: ${Breakpoint.NEW_MEETING_GRID}px)`
+const narrowScreenMediaQuery = `@media screen and (max-width: ${Breakpoint.FUZZY_TABLET}px)`
 
-const ButtonBlock = styled('div')<{isDesktop: boolean}>(({isDesktop}) => ({
+const ActionRow = styled('div')({
   alignItems: 'center',
   display: 'flex',
-  flexDirection: 'column',
-  height: '100%',
-  justifyContent: isDesktop ? 'flex-start' : 'flex-end',
-  gridArea: 'actions',
-  padding: '24px 8px',
-  width: '100%',
-  [newMeetingGridMediaQuery]: {
-    padding: '32px 8px'
-  }
-}))
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  flexWrap: 'wrap',
+  padding: 24,
+  paddingTop: 16
+})
+
+const ActiveMeetingsBlock = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'nowrap',
+  flexGrow: 10,
+  paddingTop: 8
+})
+
+const ButtonBlock = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'nowrap',
+  justifyContent: 'space-between',
+  flexGrow: 1,
+  paddingTop: 8
+})
 
-const StartButton = styled(PrimaryButton)({
-  fontSize: 18,
-  width: 320,
-  maxWidth: '100%',
-  [newMeetingGridMediaQuery]: {
-    fontSize: 24,
-    height: 64
+const StartButton = styled(FlatPrimaryButton)({
+  fontSize: 20,
+  height: 50,
+  [narrowScreenMediaQuery]: {
+    paddingLeft: '1em',
```

Comment:
-1 the em/rem/px debate goes deep, but since we don't really support browser zoom & our base size is 16px, 1em will always equal 16px, so it's nice to just use pixels so you can deal with whole numbers. otherwise if we change this we'll have to write something like `.6875em`, which gets ugly. i know there are some very old legacy components that still use em, we're trying to get rid of those!